index.js

```diff
@@ -1,16 +1,16 @@
 'use strict';
-var isFinite = require('is-finite');
+module.exports = (n, str) => {
+	str = str || ' ';
 
-module.exports = function (str, n) {
 	if (typeof str !== 'string') {
 		throw new TypeError('Expected `input` to be a string');
 	}
 
-	if (n < 0 || !isFinite(n)) {
+	if (n < 0 || !Number.isFinite(n)) {
 		throw new TypeError('Expected `count` to be a positive finite number');
```

Comment:
Should I use `typeof n` here? Because that could lead to

> Expected `count` to be a positive finite number, got `number`

Or should I use `n`

> Expected `count` to be a positive finite number, got `-4`